source/wp-content/themes/wporg-news-2021/sass/components/_local-header.scss

```diff
@@ -231,3 +220,25 @@
 		}
 	}
 }
+
+//We need this at the bottom of the file for proper cascading
+%bottom-mask {
+	position: relative;
+	background-color: transparent;
+    background: linear-gradient(to top, transparent 56px, var( --bar-background-color ) 56px 100%);
+
+	&:after {
+		content: "";
+		height: 58px;
+		position: absolute;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		z-index: -1;
+		mask-image: url('images/local-nav-mask.svg');
+		mask-repeat: no-repeat;
+		mask-position-x: -20px;
+		mask-size: cover;
```

Comment:
It looks like the image doesn't stretch infinitely, it stops at around `2000px`, so there's just negative space after that.

<img width="348" alt="Screen Shot 2021-11-26 at 9 23 03 AM" src="https://user-images.githubusercontent.com/484068/143614751-020077cb-857b-42c7-bd7d-2fa3394dd65e.png">

IIRC that was true before this PR, so feel free to open a new issue for it if you feel like it's not a quick iteration.

Reply:
yeah I think it was the max position, should be fine now